src/index.ts

```diff
@@ -31,6 +31,7 @@ try {
 const PORT = 4000
 const app = express()
 const httpServer = http.createServer(app)
+app.set('trust proxy', ['loopback', 'linklocal', 'uniquelocal'])
```

Comment:
🤔

Reply:
In GitHub Workspace with port forwarding, `req.hostname` of API requests is `localhost` when calling `https://robertu7-thematters-matters-server-hj5h-4000.githubpreview.dev/graphql`, and which isn't what we expected. 

related doc: http://expressjs.com/en/guide/behind-proxies.html

Reply:
These changes need a test on local and develop environments.